nix-alien.nix

```diff
@@ -22,7 +23,11 @@ let
             pkgs.fzf
           ];
         });
-      }
+      } // (pkgs.lib.optionalAttrs (!ci) {
+        # We want to run tests in non-CI builds but not black/mypy
+        black = pkgs.writeShellScriptBin "black" "exit 0";
+        mypy = pkgs.writeShellScriptBin "mypy" "exit 0";
```

Comment:
For the reason I am doing this instead of simply setting `doCheck = false` in non-CI builds: I still want to run tests because people can easily override the Flake inputs and so we want to run tests to ensure that nothing is broken in a random combination of packages.